symfony/plugins/orangehrmAttendancePlugin/Dao/AttendanceDao.php

```diff
@@ -130,6 +126,9 @@ public function checkForPunchOutOverLappingRecords(DateTime $punchOutTime, int $
     public function checkForPunchInOverLappingRecords(DateTime $punchInTime, int $employeeNumber): bool
     {
         $attendanceRecord = $this->getLatestAttendanceRecordByEmployeeNumber($employeeNumber);
+        if(is_null($attendanceRecord)){
+            return false;
+        }
```

Comment:
@vageeshorangehrm

Reply:
discussed this with Vageeshan and made the change

Reply:
yes we discussed this change